### ISSUE TITLE
[A11y] Explore Collections - Grid - Repeating content in main region

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1882]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

Please note right off the bat: I **do not** believe this is a hard failure.  Accessibility Insights isn't entirely clear so I think we should discuss this one.

Inside the "main" region, we include repeating content: the links for all collections, OSU collections, UO collections, "My collections", and then the filtering.

As this is a search-like interface, I do not believe the "no repeating content" rule applies, at least not in the same way as it does for pages which include site-wide links.  But it's very possible I don't understand the problem clearly.

I see three possibilities:

- This doesn't matter and can be ignored
- The links at the top of the page ("ALL" / "OREGON STATE UNIVERSITY" / etc.) are obtrusive enough that they shouldn't be in "main", but the rest is fine
- Everything that's used in searching/refining needs to be outside the "main" region

### Suggested Fix

I think a discussion is needed before determining this

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

QA depends on action taken, but likely requires at least inspecting the DOM for regions

**Comment by azle** [comment]
Updated 1/6/2024 FIXED

Updated the title to full titles.
